Asthma/Startup/APHAppDelegate.m

```diff
@@ -97,6 +98,11 @@ - (void)performMigrationAfterDataSubstrateFrom:(NSInteger) __unused previousVers
         {
             APCLogEvent(@"Migration from version %@ to %@ has failed.", [defaults objectForKey:previousVersionKey], @(kTheEntireDataModelOfTheApp));
         }
+        
+        if (![self performMigrationFromThreeToFourWithError:&migrationError])
```

Comment:
Migration code will be removed on PR#3 as new scheduler in AppCore makes it obsolete.